habitat-lab/habitat/config/default_structured_configs.py

```diff
@@ -361,7 +397,7 @@ class BaseVelocityNonCylinderActionConfig(ActionConfig):
     # is more than collision_threshold for any point.
     collision_threshold: float = 1e-5
     # The x and y locations of the clamped NavMesh position
-    navmesh_offset: Optional[List[List[float]]] = None
+    navmesh_offset: Optional[List[List[float]]] = field(default_factory=lambda:[[0.0, 0.0], [0.25, 0.0], [-0.25, 0.0]])
```

Comment:
Why do we need this navmesh offset by default?

Reply:
This is for testing other high-level policy like neural_policy and planner policy. I forget to revert it, and I will push another commit to fix it.